css/index.css

```diff
@@ -0,0 +1,11 @@
+body{
```

Comment:
Use two spaces instead of tabulation to ensure you respect the CSS best practice